generic/tests/netperf.py

```diff
@@ -148,8 +150,19 @@ def _pin_vm_threads(vm, node):
     vm.verify_alive()
     login_timeout = int(params.get("login_timeout", 360))
 
-    session = vm.wait_for_login(timeout=login_timeout)
+    server_ip = vm.wait_for_get_address(0, timeout=5)
+    if len(params.get("nics", "").split()) > 1:
+        vm.wait_for_serial_login(timeout=30, restart_network=True)
+        server_ctl = vm.wait_for_login(nic_index=1, timeout=login_timeout)
+        server_ctl_ip = vm.wait_for_get_address(1, timeout=5)
+        session = vm.wait_for_login(nic_index=1, timeout=30)
```

Comment:
What's the difference between L156 and L158? Why do you call wait_for_login twice?

Reply:
The one is session which will be end at line 189 and start before netperf testing, the other one is server_ctl which will be using during the netperf testing.